app/metrics.yaml

```diff
@@ -497,6 +497,42 @@ metrics:
     notification_emails:
       - fenix-core@mozilla.com
     expires: "2020-09-01"
+  recently_used_pwa_count:
+    type: counter
+    lifetime: application
+    description: |
+      A counter that indicates how many PWAs a user has recently used.
```

Comment:
If you've hard-coded the 30 days = recent, might want to define that here too, or maybe link to where we set that value (in case it changes)